src/surabhi/Shop.java

```diff
@@ -0,0 +1,114 @@
+package surabhi;
+
+public class Shop {
+	
+	int maggieQuantity=50, dosaQuantity=43, pouchesQuantity=39, panipuriQuantity=43, masalaQuantity=73;
+	
+	void requestedQuantity(int maggieQ, int dosaQ, int pouchesQ, int panipuriQ, int masalaQ) {
+
+		if(maggieQ<=50 && maggieQ!=0 ) {
+			maggieQuantity=maggieQuantity-maggieQ;
+			System.out.println("We are serving you Maggie " +maggieQ);
+		} else if(maggieQuantity==0){
+			System.out.println("Out of Stock");
+		} else if(maggieQ>50){
+			System.out.println("We can give "+maggieQuantity+" Maggie packets");
+			maggieQuantity=0;
+		}
+
+		if(dosaQ<=43 && dosaQ!=0) {
```

Comment:
you already give 43 value to variables then use those variables not hard coded values